jetson/odrive_bridge/src/__main__.py

```diff
@@ -197,7 +213,8 @@ def connect(self):
         global modrive
         global legal_controller
         print("looking for odrive")
-        odrives = ["2091358E524B", "20563591524B"]
+        # TODO fill in 3rd odrive id
+        odrives = ["205F3883304E", "2091358E524B", "2084399C4D4D"]

```

Comment:
can you label which drive motors each odrive controls?

Reply:
fixed